launcher/ui/MainWindow.cpp

```diff
@@ -857,8 +857,7 @@ class MainWindow::Ui
 
     void retranslateUi(MainWindow *MainWindow)
     {
-        QString winTitle = tr("%1 - Version %2", "Launcher - Version X").arg(BuildConfig.LAUNCHER_DISPLAYNAME, BuildConfig.printableVersionString());
-        MainWindow->setWindowTitle(winTitle);
+        MainWindow->setWindowTitle(tr("Instances"));
```

Comment:
> Pls feedback from other platforms

Only "Instances" appears on macOS. By the way, this is [documented by Qt](https://doc.qt.io/qt-5/qwidget.html#windowTitle-prop), but not really comprehensively:

> On some desktop platforms (including Windows and Unix), the application name (from [QGuiApplication::applicationDisplayName](https://doc.qt.io/qt-5/qguiapplication.html#applicationDisplayName-prop)) is added at the end of the window title, if set. This is done by the QPA plugin, so it is shown to the user, but isn't part of the windowTitle string.

I think it's worth noting that instances aren't really the only thing in this window. It's PolyMC's main window; it has stuff like news, buttons to settings, etc. that aren't instances (though it is definitely the main thing, so I can also see why this title makes sense).

I'm curious, what does GNOME show if you remove this line (don't set any title)? On macOS the title is "PolyMC" if you do so. (I suppose it doesn't have the version number, but none of the other apps on my computer put the version number in their window title, either, so...)

Reply:
It literally shows nothing 😅
<img width="683" alt="Screen Shot 2022-06-18 at 11 24 40 PM" src="https://user-images.githubusercontent.com/79120643/174464556-a7cdb6ce-91a5-4538-abb0-d5d1ffdeb09c.png">

Reply:
> I'd vote for calling the main window `Instances` so I don't have to figure this out.

Lol, it’s not a very serious issue anyway and pretty much just a nitpick so don’t worry if that’s what you do. I’d say if the first point below doesn’t work then just go with that option

- Does anything special happen if the window title and application display name are exactly the same? Probably not, but maybe if it’s smart it’ll just show it once
- The unified bar is already set to true. Setting it to false just changes the way the color blends with the toolbar
<img width="653" alt="49DD62FB-2FB1-4520-9C82-C159CF01F9B1" src="https://user-images.githubusercontent.com/79120643/174466549-eb50ad86-b0f2-494f-9c6c-4d0707df9f96.png">

Reply:
I also think that instances is pretty good name for this

Reply:
Your patch works on macOS too; the title is "PolyMC 1.4.0-fix/window-title"! Nice!

> Not sure if I should use `APPLICATION->applicationDisplayName()` or `QApplication::applicationDisplayName()`.

I think what you have now is fine, if only because it matches with the use of `APPLICATION` above it

Now whether you want to do this or just stick with "Instances" is up to you, my original intention with my comment was just to provide some food for thought, I never thought it would be this complicated :p